internal/platform/select_test.go

```diff
@@ -0,0 +1,90 @@
+package platform
+
+import (
+	"os"
+	"runtime"
+	"syscall"
+	"testing"
+	"time"
+
+	"github.com/tetratelabs/wazero/internal/testing/require"
+)
+
+func TestSelect(t *testing.T) {
+	if runtime.GOOS != "linux" && runtime.GOOS != "darwin" {
```

Comment:
I see you have windows implementation. do you want to make a select_test_windows.go or maybe change this expression to allow windows (then conditionally skip some parts that don't work on it?)